tests/integration/pom.xml

```diff
@@ -64,6 +64,8 @@
             <groupId>org.mule.modules</groupId>
             <artifactId>mule-module-db</artifactId>
             <version>${project.version}</version>
+            <type>test-jar</type>
```

Comment:
Instead of adding this dependency, move the AbstractConfigurationErrorTestCase class to the test functional module inside the org.mule.tck package